### PR TITLE
Only Cleanup

### PR DESCRIPTION
**Pull request by vp-stage**
cleanup && organize imports

### CODE REVIEW

**Comment by vp-stage** [comment]
Hey @lanwen, 
you could be interested by this PR.

Kind regards

File: src/test/java/ru/lanwen/wiremock/testlib/TicketEndpoint.java [hunk 1]
```diff
@@ -25,7 +22,7 @@
     public void customize(WireMockServer server, CustomizationContext customizationContext) {
         ExtensionContext context = customizationContext.getExtensionContext();
         String uuid = UUID.randomUUID().toString();
-        String testMethodName = context.getTestMethod().get().getName();
+        String testMethodName = context.getTestMethod().orElseThrow(IllegalStateException::new).getName();
```
Comment (lanwen) [inline src/test/java/ru/lanwen/wiremock/testlib/TicketEndpoint.java]: doesn't make any sense to replace one with another

File: src/main/java/ru/lanwen/wiremock/ext/WiremockResolver.java [hunk 2]
```diff
@@ -40,15 +36,13 @@ public WiremockResolver() {
     }
 
     @Override
-    public void afterEach(ExtensionContext testExtensionContext) throws Exception {
-        if (server == null || !server.isRunning()) {
```
Comment (lanwen) [inline src/main/java/ru/lanwen/wiremock/ext/WiremockResolver.java]: fail fast is preferrable

File: src/main/java/ru/lanwen/wiremock/ext/WiremockResolver.java [hunk 3]
```diff
@@ -29,7 +25,7 @@
     static final String WIREMOCK_PORT = "wiremock.port";
 
     private final WiremockFactory wiremockFactory;
-    private WireMockServer server;
+    protected WireMockServer server;
```
Comment (lanwen) [inline src/main/java/ru/lanwen/wiremock/ext/WiremockResolver.java:32]: 👍

**Review by lanwen — CHANGES_REQUESTED** [review]
star imports are banned, `final` doesn't add any value, fail fast is preferrable

**Comment by vp-stage** [comment]
Hey @lanwen,
the requested changes are done :)

**Pull request closed without merging**